### PR TITLE
Unity

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-11-11-unity-building-your-app.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+author: Valentino Urbano
+---
+
+This tutorial is going to show you how to build your app for most mobile platforms.
+
+We're going to build the applications for Android and iOS, but also for other minor platforms that Unity supports. This will not cover submitting the application to the various stores though.
+
+## Before your build
+
+This initial configuration process is common to every platform and it is done insite Unity before exporting the application.
```
Comment (MsSlacker) [inline _posts/2019-11-11-unity-building-your-app.md]: insite - > inside

File: _posts/2019-11-11-unity-building-your-app.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## Building for Tizen
+
+- Download Tizen Studio from Samsung website
+- Download the Tizen sdk when prompted
+- Link the sdk from inside Unity (inside the "Player" section)
+- Open Tizen Studio and create a signing certificate, you will need this to run the application and submit it to the store
+- Configure the Tizen cli tools as follows:
+
+I read many tutorials and none of them worked. This is what worked for me on Mac OS:
```
Comment (MsSlacker) [inline _posts/2019-11-11-unity-building-your-app.md]: This looks out of place after a :

File: _posts/2019-11-11-unity-building-your-app.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+1. Make sure you have already generated a certificate
+2. Open tizen studio and create a new native tizen application
+3. Open the terminal and cd to the folder of the project you just made
+4. Yype: `/Users/USER/tizen-studio/tools/ide/bin/tizen.sh cli-config --global "default.profiles.path=/Users/USER/workspace/.metadata/.plugins/org.tizen.common.sign/profiles.xml"` changing `/Users/USER/tizen-studio/tools/ide/bin/tizen.sh` to the shell script inside the tizen sdk folder and changing `USER` to point to your user account.
```
Comment (MsSlacker) [inline _posts/2019-11-11-unity-building-your-app.md]: Yype - > type

File: _posts/2019-11-11-unity-building-your-app.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+- Go back to unity -> PlayerSettings -> Tizen and fill the field "Signing Profile Name" with  the name of your certificate
```
Comment (MsSlacker) [inline _posts/2019-11-11-unity-building-your-app.md]: Double space after with?

File: _posts/2019-11-11-unity-building-your-app.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+- Do one last check and open `/Users/USER/tizen-studio/tools/.tizen-cli-config` and check if the config was set correctly (it happened that it said it was set and it didn't set it)
+
+Note: If you have any third party Android or iOS plugins of any kind (analytics, ads, crash reporting, ...) the build will fail. The only solution I found is to create another project without them and copy all the necessary files (without any plugin) and rebuild. There might be a better solution to it...
+
+All in all, not worth the hassle to support in my opinion, unless you really want the Asian/African market where Tizen has a big marketshare, but it was educational for sure.
+
+## Building for Samsung TV
+
+Note: If you have any third party Android or iOS plugins of any kind (analytics, ads, crash reporting, ...) the build will fail. The only solution I found is to create another project without them and copy all the necessary files (without any plugin) and rebuild. There might be a better solution to it...
```
Comment (MsSlacker) [inline _posts/2019-11-11-unity-building-your-app.md:77]: Can just do "look note for tizen"

File: _posts/2019-11-12-unity-text-in-front-of-sprite.markdown [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+title: Unity - Text in Front of 2D Sprite
+date: 2019-11-12 19:49:33.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Programming]
+author: Valentino Urbano
+---
+
+The first thought would be to have the text object be placed as children of the sprite so that it follows the sprite position as it moves around the scene. This way we avoid using the canvas and having to syncronize the position on the canvas with the position of the sprite on screen which would be a total nightmate.
```
Comment (MsSlacker) [inline _posts/2019-11-12-unity-text-in-front-of-sprite.markdown]: Umm why it starts without an introduction...? What am I reading about o.o

Comment (MsSlacker) [inline _posts/2019-11-12-unity-text-in-front-of-sprite.markdown]: syncronize - > synchronise

File: _posts/2019-11-12-unity-text-in-front-of-sprite.markdown [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+The text object needs to be of type `Text Mesh`. You can make one by creating an empty object and adding the Text Mesh as component from the inspector. It moves with the sprite as expected, unfortunately it doesn't behave well with sorting layers.
+
+I've read multiple discussions online and none of the many answers solved the problem for me. I had to combine a few of them together and add a few things to make it all work:
+
+First add a new sorting layer and place it in front of the default layer, I called it "TextElement".
+
+Note: For this method to work your text needs to be a children of the sprite so if you haven't already drag it on top of the sprite to make it a children.
```
Comment (MsSlacker) [inline _posts/2019-11-12-unity-text-in-front-of-sprite.markdown:22]: children - > child

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-12-unity-text-in-front-of-sprite.markdown (hunk 6 above)
Comment (valeIT) [reply _posts/2019-11-12-unity-text-in-front-of-sprite.markdown]: synchronise  -> synchronize

File: _posts/2019-11-11-unity-building-your-app.md (hunk 5 above)
Comment (valeIT) [reply _posts/2019-11-11-unity-building-your-app.md:77]: Not sure, would seem weird imo. Copy pasting is not the best, but it is more clear...

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**